RecoLocalCalo/HcalRecProducers/python/HBHEMethod2Parameters_cfi.py

```diff
@@ -11,7 +11,7 @@
     ts4Max                = cms.vdouble(100.,70000.), #fC # this is roughly 20 GeV
     pulseJitter           = cms.double(1.),           #GeV/bin 
     ###
-    meanTime              = cms.double(0.),   #ns 
+    meanTime              = cms.double(2.),   #ns 
```

Comment:
``` I will remove this modification (c847539) from this PR ... ``` makes sense to me.
[some possibly relevant text follows]

Is p3 (the carpet plots) made for unconstrained chi2? (the formula on the page says so, but M2 we use in reco is constrained).

The slides from HCAL DPG p6 look like the same template is used for data and MC and the template matches data very well. 
BTW, fine time binning or continuous curve overlay here would help to understand what is happening better. Due to very sharp rise of the expected pulse shape, anything significant that shows up to the left of the TS4 will very likely  trigger 3-pulse fit which will call this early entry an OOT hit that we do not record as an output of the fit. The result may give smaller bias in time, but it will give a significant bias in energy. I'd sacrifice few ns bias in time for a decreased bias in energy. Calibrations can correct the biases, but it seems to me to be more important to integrate more true signal pulse as signal to not loose resolution in E and probably loose something in t.

This "timeMean" enters in two places: 
- starting point for the fit
- constraint parameter for the fit
If MC disagrees so badly, then biases are "natural" and playing with fitting parameters without changing the expected shape is not very productive.